modules/mosdepth/main.nf

```diff
@@ -29,36 +29,17 @@ process MOSDEPTH {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
-    if (window_size) {
-        interval = "--by ${window_size}"
-    } else if ( bed ) {
-        interval = "--by ${bed}"
-    } else {
-        interval = ""
-    }
+    def reference = fasta ? "--fasta ${fasta}" : ""
+    def interval = bed ? "--by ${bed}" : ""
+
     """
     mosdepth \\
+        --threads ${task.cpus} \\
         $interval \\
+        $reference \\
         $args \\
         $prefix \\
         $bam
-    cat <<-END_VERSIONS > versions.yml
```

Comment:
What is the reason to remove the stub section?

Reply:
There was no reason to keep it, since it's not used in the tests. I wanted to clear some clutter, nothing more

Reply:
I see, I don't really know what is the plan here. I thought it could be nice that all modules have a stub section so it will be possible to perform a kind of `dry-run` of any pipeline. But as I said I don't know if this is something I only aiming for or whether this would be useful for others.

Reply:
I wasn't even aware there were plans for `dry-run` support. If so, it does make sense to have the stub.
A `dry-run` would enable us to quickly prototype workflows

Reply:
The plan is to eventually add a `stub` for all modules but we need someone to do it! It's kinda in limbo because it is only useful if it is added to all modules rather than a subset in the context of a pipeline. Quite a few should have `stub` already.

Reply:
That's good to know. Perhaps this notion should be added to the guidelines (and enforced by reviewers).